pontoon/contributors/static/js/settings.js

```diff
@@ -1,4 +1,42 @@
 $(function () {
+  // Toggle visibility
+  $('.toggle-button button').click(function (e) {
+    e.preventDefault();
+
+    var self = $(this);
+
+    if (self.is('.active')) {
+      return;
+    }
+
+    var attribute = self.data('attribute');
+    var value = self.text();
+
+    $.ajax({
+      url: '/api/v1/user/' + $('#server').data('username') + '/',
+      type: 'POST',
+      data: {
+        csrfmiddlewaretoken: $('body').data('csrf'),
+        attribute: attribute,
+        value: value,
+      },
+      success: function () {
```

Comment:
```suggestion
      success() {
```

Reply:
We use this pattern all over the place. If we were to make a change, let's do it in a separate PR.